frontend/src/pages/landingPage/landingPage.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { landingPageFg, IconDownArrow } from "../../assets/images/images";
+import { Button } from "../../components/Buttons/Buttons";
+
+function LandingPage() {
+  const LandingImage = () => {
+    return (
+      <div className="landing-image-parent">
+        <div className="svg-group-layer">
+          <img className="landing-group-image " src={landingPageFg} />
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <div className="text-center pt-5">
+      <div className="landing-top-content">
+        <h1 className="title-1 landing-header-title">
+          Together,
+          <br />
+          we can create greater civic change
+        </h1>
+        <p className="mb-5 paragraph-1 landing-head-paragraph">
+          CivicTechJobs unites ambitious technology practitioners with volunteer
+          opportunities from a central hub of listings to build digital
+          products, programs, and services.{" "}
+        </p>
+        <Button length="long" size="lg" addClass="landing-action-btn">
+          Join us
+        </Button>
+      </div>
+      <LandingImage />
+      <div className="landing-mission-section flex-container">
+        <div className="landing-mission-content px-4 align-center text-center">
+          <IconDownArrow addClass="down-arrow mb-4" />
```

Comment:
The `addClass` prop does not do anything here since IconDownArrow is not defined in our codebase (it is actually imported through svgr, under the hood, which has a limited set of props, but addClass is not one of them).

I've hinted at the solution to this being in the our documentation, but you might have missed it. The optimal approach to do this would be to import the data-url version of the svg.

```jsx
<img src={iconDownArrow} width="23px" /> // See the difference between line 15 and 19 of images.js
```

Please message me via slack on what difficulties you had encountered in finding this solution so that we may improve the documentation.